docs/development/core/api/kibana.clusterclient.md

```diff
@@ -0,0 +1,25 @@
+[Home](./index) &gt; [kibana](./kibana.md) &gt; [ClusterClient](./kibana.clusterclient.md)

```

Comment:
All of these docs mention an index file that doesn't exist, can we generate one?

Reply:
We could create one, but not sure what we would put inside it? Perhaps we can create a symlink that points to `./kibana`, then at least you don't click through to a page that doesn't exist.

Reply:
I was actually kinda hoping it would have a table of contents listing the existing docs or something like that. I think it'd be nice to see from a high level what is/isn't documented.